serfclient/connection.py

```diff
@@ -134,3 +134,14 @@ def _counter(self):
     def _error_message(self, exception):
         return "Error %s connecting %s:%s. %s." % \
             (exception.args[0], self.host, self.port, exception.args[1])
+
+    def _decode_object(self, obj_dict):
+        # Workaround for issue 20:
+        # Serf packs 'Addr' as an IP address, and can be unpack using
+        # socket.inet_ntop.
```

Comment:
I think these comments should either be removed or made more useful to those reading them. If we're going to keep them, we should make them [docstrings](https://www.python.org/dev/peps/pep-0257/#id17) and provide a link directly to the issue in question.